src/components/Card.js

```diff
@@ -1,37 +1,58 @@
-//const cardImageModal = document.querySelector("#card-image");
-//const cardPreviewImage = document.querySelector(".modal__card-image-preview");
-//const cardPreviewTitle = document.querySelector(".modal__title");
-//const cardImage = document.querySelector(".card__image");
-
 class Card {
-  constructor({ name, link }, cardSelector, handleCardClick) {
+  constructor(
+    { name, link, _id, isLiked },
+    cardSelector,
+    handleCardClick,
+    handleDeleteClick,
+    handleLikeClick
+  ) {
     this._name = name;
     this._link = link;
     this._cardSelector = cardSelector;
     this._handleCardClick = handleCardClick;
+    this._id = _id;
+    this._isLiked = isLiked;
+    this._handleDeleteClick = handleDeleteClick;
+    this._handleLikeClick = handleLikeClick;
+  }
+
+  getId() {
+    return this._id;
   }
 
   _setEventListeners() {
     this._cardElement
       .querySelector(".card__like-button")
```

Comment:
You've found the like button in line 77 as `this._likeButton`. Please use it here


```js
    this._likeButton.addEventListener("click", () => {


```


`Class fields` (the so called **class variables** with `this`) are accessible inside any method of the class. You shouldn’t pass **class fields** into the call of any method because you can use class fields inside all methods.

Reply:
completed.